### PR TITLE
launch docker from inside console

### PR DESCRIPTION
**Pull request by imikushin**
launch docker from inside console

### CODE REVIEW

**Comment by imikushin** [comment]
Comes together with rancher/os#468

File: src/docker/02-console/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,7 @@
 FROM rancher/os-base
+COPY build/ca-certificates.crt /etc/ssl/certs/
```
Comment (ibuildthecloud) [inline src/docker/02-console/Dockerfile]: This is needed any more.  Just remove this COPY

File: src/docker/02-console/docker.sh [hunk 2]
```diff
@@ -31,4 +18,4 @@ if [ -e /var/lib/rancher/conf/docker ]; then
     source /var/lib/rancher/conf/docker
 fi
 
-exec $ARGS $DOCKER_OPTS >/var/log/docker.log 2>&1
+exec /usr/bin/dockerlaunch /usr/bin/docker $ARGS $DOCKER_OPTS # >/var/log/docker.log 2>&1
```
Comment (ibuildthecloud) [inline src/docker/02-console/docker.sh]: I think we should still have this log to the file, no?

File: src/docker/03-initrdbase/.prebuild.sh [hunk 3]
```diff
@@ -7,4 +7,4 @@ mkdir -p ./build/initrd
 
 tar xf ./../00-rootfs/build/dist/rootfs-static.tar -C ./build/initrd --strip-components=2 ./bin/busybox
 tar xf ./../00-rootfs/build/dist/rootfs-static.tar -C ./build/initrd --strip-components=3 ./usr/sbin/xtables-multi
-cp ./../02-docker/build/ca-certificates.crt ./build/initrd/ca.crt
+cp ./../02-console/build/ca-certificates.crt ./build/initrd/ca.crt
```
Comment (ibuildthecloud) [inline src/docker/03-initrdbase/.prebuild.sh]: CA certs come from docker-from-scratch now.  Not needed

File: src/docker/02-console/docker.sh (hunk 2 above)
Comment (imikushin) [inline src/docker/02-console/docker.sh]: I've set stdout and stderr on the respawned process to the console container's ones, so it's visible in regular service logs

**Pull request merged**